blogango/forms.py

```diff
@@ -50,6 +50,7 @@ class PreferencesForm(forms.ModelForm):
 
     class Meta:
         model = Blog
+        fields = '__all__'
```

Comment:
@manjunath24 : "Explicit is better than implicit". Lets list the fields needed explicitly.